### PR TITLE
Pagination ticket - add 5 pages to the project with homepage css laya…

### PR DESCRIPTION
**Pull request by devdevsamuel**
…out and Next button
Screen Shot 2023-03-01 at 10 46 55 AM

### CODE REVIEW

**Review by Theweeksad — CHANGES_REQUESTED** [review]
please, use the file scheme to organize the css, as in the screenshot I sent you of the files, to have everything more organized.

File: CSS/layaout.css [hunk 1]
```diff
@@ -24,13 +24,18 @@ orden propiedades
 
     font-family: 'Noto Serif';
     font-size: 36px;
+    font-weight: 700;
+    line-height: 39.6px;
```
Comment (johanmendezb) [inline CSS/layaout.css:28]: Please move these styles to the global styles file as they should be added globally and then inherit throughout of the project.
```suggestion    
```

File: CSS/layaout.css [hunk 2]
```diff
@@ -24,13 +24,18 @@ orden propiedades
...
 }
 
 .main-subtitle {
     position: relative;
     top: 12rem;
```
Comment (johanmendezb) [inline CSS/layaout.css:33]: avoid adding space to the top, instead try to use margins to push the content to the bottom, ie: marging-bottom

File: CSS/layaout.css [hunk 3]
```diff
@@ -24,13 +24,18 @@ orden propiedades
...
     left: 3rem;
     font-size: 24px;
+    font-weight: 400;
+    line-height: 26.4px;
```
Comment (johanmendezb) [inline CSS/layaout.css:37]: Same as above, these are global styles
```suggestion
```

File: CSS/layaout.css [hunk 4]
```diff
@@ -40,9 +45,8 @@ orden propiedades
     display: flex;
     position: relative;
     top: 14rem;
-    left: 3rem;
+    left: 2em;
 
-    font-family: 'Nunito';
     font-size: 16px;
     line-height: 110%;
```
Comment (johanmendezb) [inline CSS/layaout.css]: Move them to global styles.
```suggestion
```

File: CSS/layaout.css [hunk 5]
```diff
@@ -56,25 +60,22 @@ orden propiedades
 }
 .footer a {
     color: black;
+    font-size: 16px;
+    line-height: 17.6px;
```
Comment (johanmendezb) [inline CSS/layaout.css:62]: Move them to global styles.

**Review by johanmendezb — CHANGES_REQUESTED** [review]
as a good practice all left and right margins are applied to the main/highest level container so you don't have to add it manually on each section or element on the page, for example if tomorrow, this value es not 2 but 5, you may have to look for several places to update it, instead you can just add it in one place and the change will be global, making it easier to maintain.

check `CSS/layaout.css` lines *32, 48, 69, 70*

**Pull request closed without merging**